src/modules.py

```diff
@@ -324,3 +327,17 @@ def parse_alien(config: Dict[str, Any]) -> IocCollection:
     iocs = IocCollection()
     add_url(iocs, config, "C2 alt")
     return iocs
+
+
+@module("cobaltstrike")
+def parse_cobaltstrike(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+
+    if "PublicKey" in config:
```

Comment:
we shouldn't introduce yet another specialised modules. We should solve this problem at source instead, i.e. add another "if" here:

https://github.com/CERT-Polska/mwdb_iocextract/blob/b24c0b0efb3956d155eb1cebe216a90c8bff1b35/src/modules.py#L72

(or the cobaltstrike malduck module should be changed, but it's hard with contributed code)